client/app/services/query-result.js

```diff
@@ -4,8 +4,9 @@ import { axios } from "@/services/axios";
 import { QueryResultError } from "@/services/query";
 import { Auth } from "@/services/auth";
 import { isString, uniqBy, each, isNumber, includes, extend, forOwn, get } from "lodash";
+import JSONbig from 'json-bigint';
 
-const JSONbigString = require("json-bigint")({ storeAsString: true });
+const { parse: jsonParse } = JSONbig({ storeAsString: true });
```

Comment:
And i notice at `json-bigint`https://github.com/sidorares/json-bigint?tab=readme-ov-file#optionsstoreasstring-boolean-default-false
Maybe we can remove `({ storeAsString: true })`.

Reply:
If you do this, you will convert all the values ​​​​stored by users as string or other non-numeric types into big number. This operation will drive users crazy.

Reply:
it does not work if not stored as string, I also checked the files (csv,tsv) numbers are stored as expected integers

Reply:
I will merge Andrew's PR and try with it

Reply:
It didnt work when I tried it using bigInt type as per your suggestion :/

![image](https://github.com/getredash/redash/assets/32144322/e265926e-a795-498b-8ab8-30d86633181f)

Reply:
> It didnt work when I tried it using bigInt type as per your suggestion :/
> 
> ![image](https://private-user-images.githubusercontent.com/32144322/328127227-e265926e-a795-498b-8ab8-30d86633181f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTQ5ODg1MDEsIm5iZiI6MTcxNDk4ODIwMSwicGF0aCI6Ii8zMjE0NDMyMi8zMjgxMjcyMjctZTI2NTkyNmUtYTc5NS00OThiLThhYjgtMzBkODY2MzMxODFmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MDYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTA2VDA5MzY0MVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTZjOGYzNGVhNmI1YzUzYTE0YjlkZmVlM2Q3NzU0ZDcyZDUwODY1YzU5Y2UyNzJlMTVhYTM2OGMyZjVjNGZhMzYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.hBg-JuogAEsVHTkrjFzNWe10-tzEu0KOboZ2Ro-WtZ4)

Ok, you can keep that code.